Localization/en-US/home.po

```diff
@@ -0,0 +1,110 @@
+# Schema-Version: 1.0
+# URL: https://foldingathome.org/
+# Source-Language: en-US
+# Source-Version: 1.0
+# Source-Last-Modified: 20200705
+# Target-Language:
+# Target-Version:
+# Target-Last_Modified:
+
+msgid "I AM"
+msgstr ""
+
+msgid "One"
+msgstr ""
+
+msgid "IN A MILLION"
+msgstr ""
+
+msgid "Regardless if you are already folding or haven?t heard a word about it before, we need your help to reach our goal ? which is 1 million folders."
+msgstr ""
+
+msgid "Start folding now"
+msgstr ""
+
+msgid "[Hide this and watch the video.](#)"
```

Comment:
```suggestion
msgid "Hide this and watch the video."
```

I guess this Link contains some JavaScript, which we cannot cover this way, so we'll have to exclude the link from the Markdown.